### DISCUSSION TITLE
Implementing sockets & cie to play Dofus Retro

### DISCUSSION THREAD

**Discussion by hussein-aitlahcen** [ruffle-rs/ruffle#5134] [Show and tell]
Hi guys I started hacking this project for fun last week end. I added the XMLSocket layer and had to tweak a bunch of stuff but I finally managed to make the game playable (admittedly ultra laggy after each click for unknown reasons).

Here is a video of the preview https://streamable.com/gocign

**Comment by Bale001** [comment]
Hi, do you think you could put your changes on a fork of ruffle so we can check it out? XMLSocket would be great to be added to Ruffle, even if it's not 100% accurate (I assume you would have to use WebSockets on web)

**Reply by hussein-aitlahcen** [reply]
Yes I will push of course  but currently it's a terrible piece of code I am not willing to commit as is haha! Once I have a bit of time I will make an appropriate PR
